LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2018 Yu, Li-Yu
+Copyright (c) 2022 [Tinted Theming](https://github.com/tinted-theming)
```

Comment:
```suggestion
Copyright (c) 2022 Tinted Theming (https://github.com/tinted-theming)
```

This LICENSE is not rendered as Markdown on GitHub

Reply:
:+1: Chris did this originally and I just copied that markdown style, but it seems to be plain text so doesn't make sense to use markdown formatting.